### PR TITLE
Undo the need to use quilt source format

### PR DESCRIPTION
**Pull request by jepler**
.. by giving an easy way to specify `-o root` out of the invocation of `install`. @smoe comments welcome

### CODE REVIEW

**Comment by SebKuzminsky** [comment]
I'm in favor of keeping our source format as Native instead of Quilt if possible, and managing these kinds of things in the scripts.

**Comment by smoe** [comment]
I can see where you are going. All fine with me. You may want to remove the "quilt" line and its installation. Or make this dependent on [ -d debian/patches ].
I feel a bit bad about having complicated this all for you, just cut whatever you can/want to cut, please.

**Comment by smoe** [comment]
Please add yourselves as uploaders. Other that it feels good to introduce new folks to the distribution I find it also important that you do not depend on peripheral contributors like me with your updates.

**Comment by SebKuzminsky** [comment]
@smoe please don't feel bad!  Your suggestions and efforts to help us get our software into Debian are greatly appreciated, and I fully expect there will be some bumps along the way.

**Comment by jepler** [comment]
Count me among those happy to having someone help with packaging!

File: debian/control.top.in [hunk 1]
```diff
@@ -2,7 +2,7 @@ Source: linuxcnc
 Section: misc
 Priority: optional
 Maintainer: Sebastian Kuzminsky <seb@highlab.com>
```
Comment (smoe) [inline debian/control.top.in:4]: The Maintainer should be that collective LinuxCNC address, I suggest, like you have it for mesaflash.

Comment (SebKuzminsky) [reply debian/control.top.in:4]: We've historically elected a release manager for each release cycle and put them there, but I like using the group address instead.

Comment (smoe) [reply debian/control.top.in:4]: The Maintainer gets an email when there is a bug reported and also when a new version is uploaded. I do not know enough about how that email address is used, but I can say that it is not too much traffic. Just be aware that every email you place as Maintainer or Uploader is out in the public.

**Pull request merged**